### PR TITLE
changed createmeal to function component

### PR DESCRIPTION
**Pull request by fantasticfishman**

### CODE REVIEW

**Review by raymosun — APPROVED** [review]
Looks good to me

**Comment by tusmm** [comment]
LGTM

**Comment by tusmm** [comment]
> Looks good to me

💯

File: src/components/create-meal.component.jsx [hunk 1]
```diff
@@ -0,0 +1,112 @@
+import React, { useState, useEffect, useRef } from "react";
+import "react-datepicker/dist/react-datepicker.css";
+import DatePicker from "react-datepicker";
+import axios from "axios";
+
+export default function CreateMeal(props) {
+  const [username, setUsername] = useState("");
+  const [mealName, setMealName] = useState("");
```
Comment (tusmm) [inline src/components/create-meal.component.jsx:8]: this line is very interesting

**Pull request merged**